docs/data-sources/workspace.md

```diff
@@ -33,6 +33,7 @@ resource "kubernetes_pod" "dev" {
 - `owner` (String) Username of the workspace owner.
 - `owner_email` (String) Email address of the workspace owner.
 - `owner_id` (String) UUID of the workspace owner.
+- `owner_oidc_access_token` (String) A valid OpenID Connect access token of the workspace owner. This is only available if the workspace owner authenticated with OpenID Connect. If a valid token cannot be obtained, this value will be an empty string.
```

Comment:
When implemented in the coder side, I think this should be disabled by default and enableable with a flag or at the very least the other way around.

Reply:
You should mention that in this description too.